nucliadb_ingest/nucliadb_ingest/orm/resource.py

```diff
@@ -244,7 +244,7 @@ async def generate_index_message(self) -> ResourceBrain:
                     brain.apply_field_vectors(field_key, vo, [], {})
         return brain
 
-    async def generate_broker_message(self) -> BrokerMessage:
+    async def generate_broker_message(self, cf: bool = True) -> BrokerMessage:
```

Comment:
Does `cf` mean `cloud_file`? Why not naming the parameter `cloud_file` or to something more meaningful?

Reply:
changing to `full` in order to make it more clear, it means all the bm, not the pointers